packages/core/src/Provider/StakePoolProvider/types/StakePoolProvider.ts

```diff
@@ -55,7 +65,7 @@ export interface QueryStakePoolsArgs {
   };
 
   /** Overrides default fuzzy options. Ignored in _live_ environments. */
-  fuzzyOptions?: FuzzyOptions;
+  fuzzyOptions?: PartialFuzzyOptions;
```

Comment:
```suggestion
  fuzzyOptions?: DeepPartial<FuzzyOptions>;
```
We already have a type `DeepPartial` available to us, suggest we use this instead.

Reply:
Nice to know! Thank you

Reply:
f51d482c9c should solve this